packages/typescript-configs/README.md

```diff
@@ -55,10 +54,24 @@ Here is an example of a tsconfig.json file extending the Workleap TypeScript con
 
 ```json
 {
-  "extends": "@workleap/typescript-configs/monorepo-root"
+  "extends": "@workleap/typescript-configs/monorepo-root",
+  "exclude": ["packages", "node_modules"]
 }
 ```
 
+**Note**: If you have a tsconfig.json file in your monorepo root and you exclude all packages, you might get a TS18003 error because TypeScript can't find any files to lint. To fix this, you can either add an empty `references` property to your tsconfig.json file:
+
+```json
+{
+  "extends": "@workleap/typescript-configs/monorepo-root",
+  "exclude": ["packages", "node_modules"],
+  "references": []
+}
+```
+
+You can also delete the tsconfig.json file in your monorepo root, but be aware that you will lose the ability to lint configuration files or scripts that exists at the root of the repo.
```

Comment:
Not sure I would propose this, it kind of goes against what we are trying to do. I fear someone not very familiar with TS might choose to do this.